frontend/README.md

```diff
@@ -0,0 +1,50 @@
+# React + TypeScript + Vite
```

Comment:
it would be good idea to update this autogenerated Readme with more project related info